src/verinfast/dependencies/walkers/dockerwalker.py

```diff
@@ -0,0 +1,26 @@
+from verinfast.dependencies.walkers.classes import Walker, Entry
+
+
+class DockerWalker(Walker):
+    def parse(self, file: str, expand=False):
+        with open(file, 'rb') as f:
+            string = f'\nDockerfile found {file}:\n\n\n {str(f.read())} \n\n\n'
+            self.log(timestamp=False, tag=None, msg=string)
+        with open(file, 'r') as f:
+            lines = f.readlines()
+            for line in lines:
+                line = str(line).lower()
+                if line.startswith('from'):
+                    name = line.split(' ')[1]
+                    parts = name.split(':')
+                    e = Entry(
+                        name=parts[0],
```

Comment:
Added the Dockerfile to small test repo. specifier ends up as :
`"specifier": "trusty\n",`

Reply:
Fair enough I'll strip the strings. That should fix it.

Reply:
It did. I'm also seeing the new items uploaded based on the new test files that were added. And the upload doesn't "break" tsoa type checking.
![image](https://github.com/StartupOS/verinfast/assets/78375852/8218d1cf-8e96-4d01-9018-e1137b3e6bbc)